### PR TITLE
fix: LEAP-1651: Properly update meta text for regions

### PR DESCRIPTION
**Pull request by hlomzik**
Meta text is saved in regions by Normalization mixin and it's serialized into every result of this region. When we update the meta text we should get this value again from region, not previously stored one in result.

So the priority of sources for meta value was changed. Plus "control meta" was removed.

`normInput` is also removed, it was an excess field.

### PR fulfills these requirements
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

### Describe the reason for change
Meta text was not saved during annotation update, because the old saved value took precedence in serialization.

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

It changes behaviour of meta data saving, but that's a rare feature with two data fields saved, easy to track.

### What level of testing was included in the change?
- [ ] e2e
- [ ] integration
- [ ] unit

### CODE REVIEW

File: web/libs/editor/src/components/Entity/Entity.jsx [hunk 1]
```diff
@@ -212,6 +216,7 @@ export default observer(({ store, annotation }) => {
         <Form
           style={{ marginTop: "0.5em", marginBottom: "0.5em" }}
           onFinish={() => {
+            // this component is not used in the new interface anyway
```
Comment (hlomzik) [inline web/libs/editor/src/components/Entity/Entity.jsx]: more clarification in a comment
```suggestion
            // `normInput` is undefined, but this component is not used in the new interface anyway
```

File: web/libs/editor/src/regions/Result.js [hunk 2]
```diff
@@ -319,20 +319,9 @@ const Result = types
         }
       }
 
-      const contolMeta = self.from_name.metaValue;
-
-      if (contolMeta) {
```
Comment (yyassi-heartex) [inline web/libs/editor/src/regions/Result.js:324]: im glad we're rid of `contolMeta`

**Review by yyassi-heartex — APPROVED** [review]

**Comment by MihajloHoma** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/12051716907)
> Successfully merged:  8 files changed, 67 insertions(+), 13 deletions(-)

**Review by bmartel — APPROVED** [review]
Nice cleanup and context, this will help a lot in furthering our code cleanups!

**Review by MihajloHoma — APPROVED** [review]

**Pull request merged**